examples/example_data_collection_bigym.py

```diff
@@ -0,0 +1,197 @@
+import argparse
+import time
+
+from bi_gym_utils.utils import (
+    FREQUENCY,
+    JOINT_ACTUATORS,
+    JOINT_NAMES,
+    action_to_joint_action_dict,
+    make_env,
+    obs_to_imgs,
+    obs_to_joint_dict,
+)
+from demonstrations.demo_store import DemoStore
+from demonstrations.utils import Metadata
+
+import neuracore as nc
+
+
+# --------------------------------------------------------------------------- #
+#   RUN A SINGLE EPISODE
+# --------------------------------------------------------------------------- #
+def run_episode(
+    episode_idx: int,
+    record: bool,
+    demo_store: DemoStore,
+) -> bool:
+    """Run one demonstration episode and optionally record it."""
+    print(f"\n=== Starting Episode {episode_idx} ===")
+
+    env = make_env()
+    metadata = Metadata.from_env(env)
+
+    # Get a single demo trajectory at the correct frequency
+    demo = demo_store.get_demos(metadata, amount=1, frequency=FREQUENCY)[0]
+
+    obs, info = env.reset(seed=demo.seed)
+    success = False
+    t = time.time()
+
+    try:
+        # Start neuracore recording if required
+        if record:
+            nc.start_recording()
+
+        # Log example custom metadata
+        nc.log_custom_data("my_custom_data", [1, 2, 3, 4, 5], timestamp=t)
+
+        # Initial joint + camera logging
+        qpos, qvel = obs_to_joint_dict(obs, JOINT_NAMES)
+        nc.log_joint_positions(qpos, timestamp=t)
+        nc.log_joint_velocities(qvel, timestamp=t)
+
+        images = obs_to_imgs(obs)
+        nc.log_rgb("head", images["head"], timestamp=t)
+
+        nc.log_language(
+            "Move two plates simultaneously from one draining rack to the other.",
+            timestamp=t,
+        )
+
+        # ------------------------------------------------------------------- #
+        #   Replay demonstration trajectory
+        # ------------------------------------------------------------------- #
+        for step in demo._steps:
+            # Apply demo action
+            obs, reward, terminated, truncated, info = env.step(
+                step.info["demo_action"]
+            )
+            print(
+                f"Reward={reward}, terminated={terminated}, "
+                f"truncated={truncated}, info={info}"
+            )
+
+            # Increment timestamp
+            t += 1.0 / FREQUENCY
```

Comment:
nit: set `1.0 / FREQUENCY` as a const at the top of the file. E.g. `DT = 1.0 / FREQUENCY`